product_docs/docs/pgd/5/known_issues.mdx

```diff
@@ -52,4 +52,6 @@ release.
 
 -   There currently is no protection against altering or removing a commit scope. Running transactions in a commit scope that is concurrently being altered or removed can lead to the transaction blocking or replication stalling completely due to an error on the downstream node attempting to apply the transaction. Ensure that any transactions using a specific commit scope have finished before altering or removing it.
 
+-   The [PGD CLI](cli) can return stale data on the state of the cluster if it is still connecting to nodes that have previously been parted from the cluster. Edit the [pgd-cli-config.yml](cli/installing_cli/#specifying-database-connection-strings) file to ensure only active nodes in the cluster are listed for connection.
```

Comment:
I think this should also go [here](https://www.enterprisedb.com/docs/pgd/latest/cli/installing_cli/#using-a-configuration-file) because this is kind of expectation rather than limitations. 

There is also `--dsn` [flag](https://www.enterprisedb.com/docs/pgd/latest/cli/installing_cli/#using-a-configuration-file).